.github/workflows/reusable-toolbox-tests.yml

```diff
@@ -26,10 +26,10 @@ jobs:
           cache: yarn
       - name: Install dependencies
         run: yarn install --frozen-lockfile
-      # To fail fast, build the Toolbox to see if any errors occur. For example, a common error is building referenced projects like the introspector.
-      - name: Build the GraphQL Toolbox
+      # the introspector is a referenced project by the Toolbox.
```

Comment:
```suggestion
      # the introspector is referenced project by the Toolbox.
```

Reply:
Did you mean `the introspector is referenced by the Toolbox.`? Or do I misunderstand your suggestion?

Reply:
My mistake!